.github/workflows/code-references.yml

```
@@ -0,0 +1,40 @@
+---
+name: Collect code references
+
+on:
+  pull_request:
+
```

```suggestion
on:
  pull_request:
    branches-ignore:    
        - "renovate/**"
```

I would recommend disabling for Renovate unless it's needed.

---

It's not needed. Do you think this will be the norm for some workflows? Any plans to roll out elsewhere?

---

I believe it should, no plans yet, but I have been keeping my eyes out for new workflows and trying to verify when I can.